recipes/basemap-data-hires/meta.yaml

```diff
@@ -1,4 +1,4 @@
-package:
+ package:
```

Comment:
Nevermind, we need to fix it. Linter [failed]( https://travis-ci.org/conda-forge/staged-recipes/jobs/118524074#L300 ).